docs/githubteacher.md

```diff
@@ -0,0 +1,7 @@
+# GitHub Teacher Bio
+
+**Name:** GitHub Teacher
+
+**Occupation:** Teacher
+
+**Location:** San Francisco
```

Comment:
I think we might want to be more specific here.  There are lots of San Franciscos in the :earth_americas: